macros/sql/surrogate_key.sql

```diff
@@ -0,0 +1,16 @@
+{% macro surrogate_key(fields) -%}
```

Comment:
You can leave out this `fields` argument and then use `varargs` instead of `fields` down below. The `varargs` magic variable becomes a list of arguments passed to this macro, so you can drop the `fields=[...]` boilerplate and write

```
{{ dbt_utils.surrogate_key('field_a', 'field_b') }}
```

Kind of personal preference here, so totally fair if you prefer the list syntax. I like that the varargs syntax feels similar to `coalesce`.

Reply:
i agree with you. I didn't know about `varargs` but just updated. very easy.